Stylesheets/Components/avatar.css

```diff
@@ -0,0 +1,45 @@
+.avatar-circle {
+    max-width: 100%;
+    height: auto;
+    border-radius: 50%;
+    width: 2.5rem;
+}
+
+.avatar-square {
+    max-width: 100%;
+    height: auto;
+    border-radius: 10%;
+    width: 2.5rem;
+}
+
+.avatar img {
+    height: 2.5rem;
+    width: 2.5rem;
+}
+
+.avatar-shadow {
+    box-shadow: 10px 10px 2px 10px #888888;
```

Comment:
use color variables wherever possible

Reply:
Thanks for pointing it out vanshita. Will change it